File: 04/ABC089B/ans-yellow.py
# 入力
n = int(input())
s = input().split()

# 黄色があるかどうかを格納する変数
yellow = False

# 各色をチェックする
for i in range(n):
    if s[i] == "Y":
        yellow = True
    # ほかの色は、必ず存在することが保証されているので、無視してよい

# 黄色があればFour
if yellow:
    print("Four")
else:
    print("Three")

File: 04/ABC089B/ans-yellow.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  // 入力
  int n;
  cin >> n;

  // 黄色があるかどうかを格納する変数
  bool yellow = false;
  for (int i = 0; i < n; i++) {
    string s;
    cin >> s;

    if (s == "Y") {
      yellow = true;
    }
    // ほかの色は、必ず存在することが保証されているので、無視してよい
  }

  // 黄色があるなら、Four
  // そうでないなら、Three
  cout << (yellow ? "Four" : "Three") << endl;
}